### ISSUE TITLE
some Twitter profiles not linking

### ISSUE THREAD

**Issue by dschep** [dschep/proven#21]
https://keybase.io/jonnu

**Comment by NadyaNayme** [comment]
The profile works just fine for me in Firefox - so this may be Chrome specific. I'm at home now, so can't test Chrome, sorry! I tried stepping through with a debugger to see if I could help with this - so just leaving my report that "FF (61.0.2) looks fine for me" on that users profiles (all of them work!)

**Comment by davidpomerenke** [comment]
Does not work for me on Twitter either, using Firefox. The screenshots in the readme show the old layout of Twitter, maybe the add-on has not yet adapted to the new twitter layout? Afaik, it is now the default layout and I don't see an option to switch to the old layout to test whether it works for me there.

**Comment by dschep** [comment]
yeah new layout isn't supported yet. mobile is broken too currently. weird classnames make it a pain to update :(
